src/app/shared/guards/is-case-allocator.guard.ts

```diff
@@ -0,0 +1,16 @@
+import { select, Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import * as fromActions from '../../store';
```

Comment:
Sort imports alphabetically, Otherwise happy to approve

Reply:
seems right to me